test/FastTests/Client/Indexing/IndexesFromClient.cs

```diff
@@ -148,41 +148,43 @@ public class IndexesFromClient : RavenTestBase
                 var index = indexes[0];
                 WaitForIndexing(store);
 
-                var stats = store.Admin.Send(new GetIndexStatisticsOperation(index.Name));
-                try
+                IndexStats stats = null;
+                Assert.True(SpinWait.SpinUntil(() =>
```

Comment:
I think that we had a race here, stats are written after indexing run, so IsStale could be false, but stats were not yet updated.

Reply:
Yes, but this is the case even if we wait, because the stats are updated from multiple locations in different times